app/Enums/SettlementType.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Enums;
+
+enum SettlementType: string
+{
+    case Hours = "hours";
+    case Money = "money";
+
+    public function label(): string
+    {
+        return __($this->value);
+    }
+
+    public static function casesToSelect(): array
+    {
+        $cases = collect(SettlementType::cases());
+
+        return $cases->map(
+            fn(SettlementType $enum): array => [
+                "label" => $enum->label(),
+                "value" => $enum->value,
+            ],
+        )->toArray();
```

Comment:
Isn't that overkill?

```suggestion
        return array_map(
            fn(SettlementType $enum): array => [
                "label" => $enum->label(),
                "value" => $enum->value,
            ],
            SettlementType::cases(),
        )
```